api/agent/runner_client.go

```diff
@@ -247,14 +252,17 @@ func sendToRunner(ctx context.Context, protocolClient pb.RunnerProtocol_EngageCl
 		// WARNING: blocking read.
 		n, err := bodyReader.Read(writeBuffer)
 		if err != nil && err != io.EOF {
-			log.WithError(err).Error("Failed to receive data from http client body")
+			errorMsg = fmt.Sprintf("Failed to receive data from http client body")
```

Comment:
This is a const-string, no need for 'fmt.Sprintf'

Reply:
Good eye, will change that back.